dev/Scroller/TestUI/ScrollerDynamicPage.xaml

```diff
@@ -124,7 +124,6 @@
                 <TextBlock Text="HorizontalScrollMode:" Grid.Row="3" Grid.Column="0" VerticalAlignment="Center"/>
                 <ComboBox x:Name="cmbHorizontalScrollMode" Margin="1" Grid.Row="3" Grid.Column="1" 
                       HorizontalAlignment="Stretch" VerticalAlignment="Center" SelectionChanged="CmbHorizontalScrollMode_SelectionChanged">
-                    <ComboBoxItem>Auto</ComboBoxItem>
```

Comment:
Should this be present if USE_SCROLLMODE_AUTO is true?

Reply:
Yes - I did not spend much time trying to figure out how to do that in XAML markup. I thought the easiest thing to do would be to put it back if we decide to resurrect Auto. It would be hard to miss it in that event.

Reply:
Hmm, I don't know if you can do that from markup, but you could remove the combo box item from the combo box in the page loaded event when the compiler flag is not set. (Or add it when it is set).  Either is messy so leaving it out might be okay, but it means you can't simply set the flag and rebuild as it is.

Reply:
I think it's OK to have to add this line back (among other few) in the event the Auto value comes back.